public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,239 @@
+                  - example:
+                      code: ACCEPT_HEADER_INVALID
+              examples:
+                badRequest:
+                  description: The accept header is missing or invalid
+                  value:
+                    code: ACCEPT_HEADER_INVALID
+      deprecated: false
+      security:
+        - userRestricted:
+            - common-transit-convention-guarantee-balance
+components:
+  schemas:
+    errorResponse:
+      title: errorResponse
+      required:
+        - code
+      type: object
+      properties:
+        code:
+          type: string
+    hal-document-schema.json:
+      title: hal-document-schema.json
+      type: object
+      properties:
+        _links:
+          allOf:
+            - $ref: '#/components/schemas/Links'
+            - description: An object containing links to other resources
+        _embedded:
+          type: object
+          additionalProperties: {}
+          description: An object containing embedded resources
+      description: A HAL document
+    CustomsGuaranteesBalancesrequest:
+      title: CustomsGuaranteesBalancesrequest
+      required:
+        - accessCode
+      type: object
+      properties:
+        accessCode:
+          maxLength: 4
+          minLength: 4
+          type: string
+          description: The access code granting access to make use of the guarantee.
+      description: A balance request for the CTC Guarantee Balance API
+    GetBalanceOk:
+      title: Balance request successful
+      required:
+        - balance
+      type: object
+      properties:
+        _links:
+          type: object
+          additionalProperties:
+            $ref: '#/components/schemas/halLink'
+          description: An object containing links to other resources.
+        _embedded:
+          type: object
+          additionalProperties:
+            $ref: '#/components/schemas/hal-document-schema.json'
+          description: An object containing embedded resources.
+        balance:
+          description: The guarantee's balance
+          type: number
+      description: A successful balance response.
+    BadRequest:
+      title: BadRequest
+      required:
+        - code
+        - message
+      type: object
+      properties:
+        code:
+          enum:
+            - BAD_REQUEST
+          type: string
+        message:
+          type: string
+          description: The description of the error.
+      description: An error in the supplied payload, or the Guarantee Reference Number did not match the expected format
+    NotFound:
+      title: NotFound
+      required:
+        - code
+        - message
+      type: object
+      properties:
+        code:
+          enum:
+            - NOT_FOUND
+          type: string
+        message:
+          type: string
+          description: The description of the error.
+      description: A guarantee was not found that matched the supplied Guarantee Reference Number and Access Code
```

Comment:
For ease of readability and maintenance, I would move this description immediately under `title`.

Add a full stop.